File: src/main/webapp/app/model/Book.js
Ext.define('Search.model.Book', {
	extend : 'Ext.data.Model',
	fields : [ {
		name : 'id',
		type : 'int'
	}, {
		name : 'title',
		type : 'string'
	}, {
		name : 'author',
		type : 'string'
	}, {
		name : 'price',
		type : 'int'
	}, {
		name : 'qty',
		type : 'int'
	}, {
		name : 'title_author',
		type : 'string',
		convert : function(v, record) {
			return record.get('title') + " (" + record.get('author') + ")";
		}
	} ]
});

File: src/main/java/org/search/model/Book.java
package org.search.model;

public class Book {

	private int id;
	private String title;
	private String author;
	private int price;
	private int qty;

	public Book() {
	}

	public Book(int id, String title, String author, int price, int qty) {
		this.id = id;
		this.title = title;
		this.author = author;
		this.price = price;
		this.qty = qty;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getAuthor() {
		return author;
	}

	public void setAuthor(String author) {
		this.author = author;
	}

	public int getPrice() {
		return price;
	}

	public void setPrice(int price) {
		this.price = price;
	}

	public int getQty() {
		return qty;
	}

	public void setQty(int qty) {
		this.qty = qty;
	}

	@Override
	public int hashCode() {
		final int prime = 31;
		int result = 1;
		result = prime * result + id;
		return result;
	}

	@Override
	public boolean equals(Object obj) {
		if (this == obj) {
			return true;
		}
		if (obj == null) {
			return false;
		}
		if (!(obj instanceof Book)) {
			return false;
		}
		Book other = (Book) obj;
		if (id != other.id) {
			return false;
		}
		return true;
	}

	@Override
	public String toString() {
		return String.format("Book [title = %s, author = %s]", title, author);
	}
}